src/transaction.js

```diff
@@ -115,6 +115,9 @@ Transaction.fromBuffer = function (buffer, __noStrict) {
     }
     tx.locktime = readUInt32()
     tx.exprie = readUInt32()
+  }
+
+  if (tx._stype === Transaction.TxSerializeFull) {
```

Comment:
it's incorrect, the `timestamp` is for ``` tx._stype === Transaction.TxSerializeFull ||
    tx._stype === Transaction.TxSerializeNoWitness) ```, has already done at line 99

Reply:
timestamp Only the hashFull is needed, OnlyWitness is not needed